docs/_articles/en/v3.x/ci-cd-integration.md

```diff
@@ -19,7 +19,29 @@ Follow these CI/CD best practices
 * Use the `-o/--outfile=<name>.xml` flag to write your results to a file and produce a results artifact that can be used by the CI/CD tool.
 * Use the `-s/–severity-threshold` flag to cause a non-zero exit code when any violations meet or exceed the provided value. Many CI/CD tools require thresholds.
 
-## Accelerate Your CI/CD Integration
+## Accelerate Your CI/CD Integration #
```

Comment:
@stephen-carter-at-sf Just FYI, I learned this from Josh & Roopa: the trailing # allows the text to become an anchor link, so that we can link to this subtopic. So the trailing # is intentional. Thanks for asking!